### PR TITLE
feat(livestock-animal-set-join): animal set join documentation

### PR DESCRIPTION
**Pull request by ryanjosephtorrado**
Ticket: [AB#19791](https://dev.azure.com/rezaresystems/15020b92-fee6-4202-956f-9036d1e1da84/_workitems/edit/19791)

----

Notes
- adds the `livestock-animal-set-join` documentation

----

## PR Checklist
- [ ] Documentation has been checked by at least one team member
- [x] Links work to navigate documentation
- [ ] Images stored in `_images`
- [x] Spell checked

> If some of the above are not completed why?
> **REASON HERE**

### CODE REVIEW

File: docs/resource-types/livestock/animal-set-join.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+# Resource Type: Livestock - Animal Set Joins
+
+- [URLs](#urls)
+- [Response Structure](#response-structure)
+
+---
+
+## URLs
+
+Get all animal set join that you have access to
```
Comment (markusPirkl) [inline docs/resource-types/livestock/animal-set-join.md:23]: I think it is joins. So add a `s` please

File: docs/resource-types/livestock/animal-set-join.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+## Response Structure
+
+A call to the animal set join endpoints returns the following fields.
+
+```json
+{
+  "@self": "string",
+  "location": {...},
+  "meta": {...},
+  "resourceType": "/livestock/animal-set",
```
Comment (markusPirkl) [inline docs/resource-types/livestock/animal-set-join.md:58]: This should probably be `/livestock/animal-set-join`?

File: docs/resource-types/livestock/animal-set-join.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  "remark": "string",
+  "reponsible": "string",
+  "traitLabel": {...},
+  "animal": {...},
+}
+```
+
+| Response Item          | Description                                                                                                                 | Data Type                                          |
+| ---------------------- | --------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------- |
+| **Self**               | A link to this specific animal set                                                                                          | URI                                                |
```
Comment (markusPirkl) [inline docs/resource-types/livestock/animal-set-join.md:71]:
```suggestion
| **Self**               | A link to this specific animal set                                                                                          | URI                                                |
```
| **Self**               | A link to this specific animal set  join                                                                                        | URI                                                |

File: docs/resource-types/livestock/animal-set-join.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+| **Location**           | An identifier for the location of the animal set                                                                            | [Identifier](/resource-types/common.md#identifier) |
```
Comment (markusPirkl) [inline docs/resource-types/livestock/animal-set-join.md:72]:
```suggestion
| **Location**           | An identifier for the location of the animal set join                                                                           | [Identifier](/resource-types/common.md#identifier) |
```

File: docs/resource-types/livestock/animal-set-join.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+| **Meta**               | Meta data for the resource                                                                                                  | [Metadata](/resource-types/common.md#metadata)     |
+| **Resource Type**      | The fixed discriminator for the animal set join resource type.<br/>Value: /livestock/animal-set -join                       | String                                             |
+| **Contemporary Group** | For manually recorded events, record any contemporary group code that would affect statistical analysis                     | String                                             |
+| **Event Date Time**    | A particular point in the progression of time                                                                               | Date/Time                                          |
```
Comment (markusPirkl) [inline docs/resource-types/livestock/animal-set-join.md:76]: This is clearly not what we want in the dev hub docs. :D

How about the date and time when the animal joined the set?

**Review by markusPirkl — DISMISSED** [review]
see comments

File: docs/resource-types/livestock/animal-set-join.md (hunk 2 above)
Comment (ryanjosephtorrado) [reply docs/resource-types/livestock/animal-set-join.md:58]: oh, right! good eye. Thanks Markus

**Comment by ryanjosephtorrado** [comment]
> see comments

@markusPirkl, I've updated the description. Please check and review. Thank you

File: docs/resource-types/livestock/animal-set-join.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+  "traitLabel": {...},
+  "animal": {...},
+  "animalSetId": "string"
+}
+```
+
+| Response Item          | Description                                                                                                | Data Type                                          |
+| ---------------------- | ---------------------------------------------------------------------------------------------------------- | -------------------------------------------------- |
+| **Self**               | A link to this specific animal set join event                                                              | URI                                                |
+| **Location**           | An identifier for the location of the animal set join event                                                | [Identifier](/resource-types/common.md#identifier) |
```
Comment (chriiiish) [inline docs/resource-types/livestock/animal-set-join.md:73]:
```suggestion
| **Location**           | An identifier for the location where the animal set join event occurred                      | [Identifier](/resource-types/common.md#identifier) |
```

File: docs/resource-types/livestock/animal-set-join.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+| **Meta**               | Meta data for the resource                                                                                 | [Metadata](/resource-types/common.md#metadata)     |
+| **Resource Type**      | The fixed discriminator for the animal set join event resource type.<br/>Value: /livestock/animal-set-join | String                                             |
+| **Contemporary Group** | A contemporary group code that would affect statistical analysis                                           | String                                             |
+| **Event Date Time**    | The time the event occurred                                                                                | Datetime                                           |
```
Comment (chriiiish) [inline docs/resource-types/livestock/animal-set-join.md:77]:
```suggestion
| **Event Date Time** | The date and time the event occurred | Datetime |
```

**Review by chriiiish — DISMISSED** [review]
Some final wording suggestions, but nothing major

**Review by chriiiish — APPROVED** [review]

**Pull request merged**